Add When to use a Symfony Messenger component (#45)
bornfight/README: playbooks/symfony/README.md

<!-- prettier-ignore-start -->
<!-- start_toc -->
| Title | Overview |
|---|---|

> | [When to use a Symfony Messenger component 🚚](/playbooks/symfony/symfony-messenger.md#readme) | A brief guideline on when is the right time to use a Symfony Messenger component |

Reviewer: ```suggestion
| [When to use a Symfony Messenger component 🚚](/playbooks/symfony/symfony-messenger.md#readme) | A brief guideline on situations where Symfony Messenger component could be used |
```
